app/controllers/text_components_controller.rb

```diff
@@ -124,7 +141,7 @@ def text_component_params
       params.require(:text_component)
         .permit(:heading, :introduction, :main_part, :closing, :from_day, :to_day,
                 :timeframe, :to_hour, :report_id, :topic_id, :assignee_id, :image, :image_alt, :delete_image,
-                :publication_status, :notes, trigger_ids: [], channel_ids: [],
+                :publication_status, :notes, :duplicate, trigger_ids: [], channel_ids: [],
```

Comment:
You probably don't need to include the `:duplicate` here as it is not an attribute that will go into the `TextComponent`.

Reply:
Oh, that was actually leftover from my initial approach passing an existing text component’s id to the `new` action…